.changeset/breezy-cats-grab.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': patch
+'@astrojs/markdoc': patch
+'@astrojs/mdx': patch
+---
+
+Introduce the (experimental) `@astrojs/markdoc` integration. This unlocks Markdoc inside your Content Collections, and brings support for Astro and UI components in your content. This also improves internals to make Content Collections extensible to more file types in the future.
```

Comment:
can you expand this changeset a little bit? Like maybe show the `astro add` command and/or show the config diff that adds this.

Reply:
Done! Included links to where the docs and `with-markdoc` starter will go as well